pyfar/dsp/dsp.py

```diff
@@ -1386,6 +1386,232 @@ def time_shift(signal, shift, unit='samples'):
     return shifted.reshape(signal.cshape)
 
 
+def find_impulse_response_delay(ir, N=1):
+    """Find the delay in samples of an impulse response.
+    The method relies on the analytic part of the cross-correlation function
```

Comment:
Add a blank line here?
```suggestion
    
    The method relies on the analytic part of the cross-correlation function
```